domain/model/modelmigration/import.go

```diff
@@ -289,7 +297,7 @@ func (i *importModelOperation) Execute(ctx context.Context, model description.Mo
 	}
 
 	// We need to establish the read only model information in the model database.
-	err = i.modelDetailServiceFunc(modelID).CreateModelForVersion(ctx, controllerUUID, args.AgentVersion)
+	err = i.modelDetailServiceFunc(modelID).CreateModelForVersion(ctx, controllerUUID, args.AgentVersion, agentStream)
```

Comment:
Are we going to do the export in a following PR?

Reply:
yes, the export will still happen through model config as we are not going to do wide sweeping changes through the client and description to break this out of model config completely.